generated/index.d.ts

```diff
@@ -1288,12 +1290,12 @@ interface GPUBindingCommandsMixin {
    */
   setBindGroup(
     index: GPUIndex32,
-    bindGroup: GPUBindGroup,
+    bindGroup: GPUBindGroup | null,
```

Comment:
These changes seem to be missing from dist/?

Reply:
Oops, added back. I think I was looking at an older tab of the spec and I didn't see the update on the nullables so I thought it was a generator artifact or something. Thanks for catching it!